kong/information.go

```diff
@@ -0,0 +1,53 @@
+package kong
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+)
+
+type Info struct {
+	Version       string `json:"version,omitempty" yaml:"version,omitempty"`
+	Configuration struct {
+		Database string `json:"database,omitempty" yaml:"database,omitempty"`
+		Portal   bool   `json:"portal,omitempty" yaml:"portal,omitempty"`
+		RBAC     string `json:"rbac,omitempty" yaml:"rbac,omitempty"`
+	} `json:"configuration,omitempty" yaml:"configuration,omitempty"`
+}
+
+// IsEnterprise check if Kong is enterprise edition
+func (s *Info) IsEnterprise() bool {
+	return strings.Contains(s.Version, "enterprise")
+}
+
+// IsInMemory check if Kong is in memory
+func (s *Info) IsInMemory() bool {
+	return "off" == s.Configuration.Database
+}
+
+// IsRBACEnabled check if RBAC are enabled
+func (s *Info) IsRBACEnabled() bool {
+	return "on" == s.Configuration.RBAC
+}
+
+// IsPortalEnabled check if the portal is enabled
+func (s *Info) IsPortalEnabled() bool {
+	return s.Configuration.Portal
+}
```

Comment:
`s.IsPortalEnabled()` is equivalent to `s.Configuration.Portal`. There's no need for this function to exist if the user can just read the bool from the public field.

Suggestion: remove